src/Postmark/PostmarkAdminClient.cs

```diff
@@ -246,6 +246,87 @@ protected override string AuthHeaderName
                ("/senders/" + signatureId, HttpMethod.Put, parameters);
         }
 
+        /// <summary>
+        /// Get a list of domains.
+        /// </summary>
+        /// <param name="offset"></param>
+        /// <param name="count"></param>
+        /// <returns></returns>
+        public async Task<PostmarkDomainList> GetDomainsAsync(int offset = 0, int count = 100)
+        {
+            var parameters = new Dictionary<string, object>();
+            parameters["offset"] = offset;
+            parameters["count"] = count;
+            return await this.ProcessNoBodyRequestAsync<PostmarkDomainList>("/domains", parameters);
+        }
+
+
+        /// <summary>
+        /// Retrieve a domain.
+        /// </summary>
+        /// <param name="domainId"></param>
+        /// <returns></returns>
+        public async Task<PostmarkCompleteDomain> GetDomainAsync(int domainId)
+        {
+            return await this.ProcessNoBodyRequestAsync<PostmarkCompleteDomain>("/domains/" + domainId);
```

Comment:
Your call, @jhuggart, but I'd be cool with us using the C# 6 string interpolation for the request paths instead of string concatenation. May be a little easier to read.